src/core/CustomData.cpp

```diff
@@ -180,7 +193,7 @@ void CustomData::updateLastModified(QDateTime lastModified)
         return;
     }
 
-    if (!lastModified.isValid()) {
+    if (!lastModified.isValid() && canUpdateTimeinfo()) {
```

Comment:
oh this doesn't actually make sense, if it's not valid we shouldn't insert it.

OH! WOW, why is it not valid!?!!?!?!?!?!

I take make this whole PR.

Reply:
is the browser extension putting in invalid dates!?!?!?!?!?!

Reply:
is `lastModified` supposed to be a method call?